requirements.txt

```diff
@@ -0,0 +1,89 @@
+astroid==1.3.8
+Babel==2.0
+cliff==1.13.0
+cmd2==0.6.8
+cookies==2.2.1
+coverage==3.7.1
+debtcollector==0.6.0
+dj-static==0.0.6
+Django==1.8.3
+django-angular==0.7.15
+django-appconf==1.0.1
+django-compressor==1.5
+django-debug-toolbar==1.3.2
+django-environ==0.3.0
+-e git+https://github.com/django-extensions/django-extensions.git@45393942002ca04c4dc00b8915fa4e75fb48c64e#egg=django_extensions-dev
+django-filter==0.10.0
+django-grappelli==2.7.1
+django-libsass==0.3
+django-redis==4.2.0
+django-zurb-foundation==5.5.0
+djangorestframework==3.2.1
+dodgy==0.1.7
+factory-boy==2.5.2
+freezegun==0.3.5
```

Comment:
@antoviaque That makes sense. I had no particular reason I guess, just a preference for installing fewer packages where possible, since complex dependency graphs tend to get hard to manage.